go.mod

```diff
@@ -1,20 +1,34 @@
 module github.com/cofide/cofide-demos
 
-go 1.22.5
+go 1.23.3
 
-toolchain go1.22.8
+toolchain go1.24.0
 
-require github.com/spiffe/go-spiffe/v2 v2.3.0
+require github.com/spiffe/go-spiffe/v2 v2.5.0
 
 require (
 	github.com/Microsoft/go-winio v0.6.2 // indirect
+	github.com/cofide/cofide-sdk-go v0.0.0-00010101000000-000000000000
 	github.com/go-jose/go-jose/v4 v4.0.5 // indirect
-	github.com/zeebo/errs v1.3.0 // indirect
+	github.com/zeebo/errs v1.4.0 // indirect
 	golang.org/x/crypto v0.32.0 // indirect
-	golang.org/x/net v0.33.0 // indirect
+	golang.org/x/net v0.34.0 // indirect
 	golang.org/x/sys v0.29.0 // indirect
 	golang.org/x/text v0.21.0 // indirect
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20240604185151-ef581f913117 // indirect
-	google.golang.org/grpc v1.64.1 // indirect
-	google.golang.org/protobuf v1.35.1 // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20250124145028-65684f501c47 // indirect
+	google.golang.org/grpc v1.70.0 // indirect
+	google.golang.org/protobuf v1.36.5 // indirect
 )
+
+require (
+	cel.dev/expr v0.19.0 // indirect
+	github.com/cncf/xds/go v0.0.0-20240905190251-b4127c9b8d78 // indirect
+	github.com/envoyproxy/go-control-plane v0.13.4 // indirect
+	github.com/envoyproxy/go-control-plane/envoy v1.32.3 // indirect
+	github.com/envoyproxy/protoc-gen-validate v1.1.0 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
+	github.com/planetscale/vtprotobuf v0.6.1-0.20240319094008-0393e58bdf10 // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20250212204824-5a70512c5d8b // indirect
+)
+
+replace github.com/cofide/cofide-sdk-go => ../cofide-sdk-go
```

Comment:
note: a local override for dev, to be replaced when we release `cofide-sdk-go` with the experimental XDS capability.

Reply:
Replaced with `cofide-sdk-go@v0.2.0`